src/ApcTokenStorage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace F3\AppNexusClient;
+
+/**
+ * ApcTokenStorage
+ *
+ * @uses TokenStorage
+ * @package
+ * @version $id$
+ * @copyright Federico Nicolás Motta
+ * @author Federico Nicolás Motta <fedemotta@gmail.com>
+ * @license http://opensource.org/licenses/mit-license.php The MIT License (MIT)
+ */
+class ApcTokenStorage implements TokenStorage
+{
+    private $prefix;
+    private $ttl;
+
+    public function __construct($prefix = '',  $ttl = 0 )
+    {
```

Comment:
I suggest

        if ( ! extension_loaded('apc')) {
            throw new \RuntimeException('APC extension is not loaded');
        }